src/main/java/uk/gov/justice/hmpps/prison/api/resource/AdjudicationsResource.java

```diff
@@ -41,6 +42,23 @@ public AdjudicationsResource(final AdjudicationsService adjudicationsService) {
         this.adjudicationsService = adjudicationsService;
     }
 
+    @ApiResponses({
+        @ApiResponse(responseCode = "201", description = "Created"),
+        @ApiResponse(responseCode = "400", description = "Invalid request - e.g. no current booking was found for the given offender.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))}),
+        @ApiResponse(responseCode = "404", description = "No match was found for the provided offender number.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})
+    })
+    @Operation(summary = "Request the information needed to create an adjudication.",
+        description = "Must be called before creating the adjudication. Requires MAINTAIN_ADJUDICATIONS access and write scope")
+    @PostMapping("/adjudication/request-creation-data")
+    @ProxyUser
+    @PreAuthorize("hasRole('MAINTAIN_ADJUDICATIONS') and hasAuthority('SCOPE_write')")
+    public ResponseEntity<AdjudicationCreationRequestData> requestAdjudicationCreationData(@RequestBody final String offenderNo) {
```

Comment:
Very minor but If you prefer you could just return a AdjudicationCreateRequestData and annotate with a 201 response code